layouts/page/nx-cugraph.html

```diff
@@ -0,0 +1,325 @@
+{{- define "main" -}}
+
+<!-- Hero Partial -->
+{{ partial "hero.html" (dict
+"text" "GPU Accelerated NetworkX Backends"
+"btn_text" "Try Now"
```

Comment:
Instead of "Try Now" linking to the GitHub, we should do what `cudf-pandas` does where, if you click "Get Started", it takes you straight down to the call-to-action at the bottom. 

![image](https://github.com/user-attachments/assets/fa6a6299-df89-4759-9097-3472c287633b)

Reply:
Resolved in #419